packages/web-components/src/components/va-date/va-date.tsx

```diff
@@ -67,21 +127,40 @@ export class VaDate {
     if (target.classList.contains('input-year')) {
       currentYear = target.value;
     }
-    // Use a leading zero for numbers < 10
-    const numFormatter = new Intl.NumberFormat('en-US', {
-      minimumIntegerDigits: 2,
-    });
 
-    // If Month or Date Select are chosen as empty return ''
-    // Otherwise convert numbers less than 10 to have a leading 0
-    this.value = `${currentYear}-${
-      currentMonth ? numFormatter.format(parseInt(currentMonth)) : ''
-    }-${currentDay ? numFormatter.format(parseInt(currentDay)) : ''}`;
+    this.value = `${currentYear}-${currentMonth ? currentMonth : ''}-${
+      currentDay ? currentDay : ''
```

Comment:
Since the current year, month, & day are coming from `target.value`, in what situations would they be falsey? All these ternaries make the expression harder to read, and I'm wondering if we could get away with something like:

```js
this.value = `${currentYear}-${currentMonth}-${currentDay}`;
```

Reply:
Yes you are correct and I have updated accordingly to fix this issue.